packages/core/src/routes/user-assets.ts

```diff
@@ -52,13 +52,15 @@ export default function userAssetsRoutes<T extends ManagementApiRouter>(
     '/user-assets',
     koaGuard({
       files: object({
-        file: uploadFileGuard,
+        file: uploadFileGuard.array(),
       }),
       response: userAssetsGuard,
     }),
     async (ctx, next) => {
-      const { file } = ctx.guard.files;
+      const { file: bodyFiles } = ctx.guard.files;
 
+      const file = bodyFiles[0];
+      assertThat(file, 'guard.file_not_found');
```

Comment:
how about, letting the zod to guard the file length? Just throw guard.invalid_input here as a callback error

Reply:
This is still needed to make TS happy

Reply:
updated